### PR TITLE
Fix more tests

### PR DESCRIPTION
**Pull request by adriangb**
Selected Reviewer: @dmontagu

### CODE REVIEW

File: tests/test_dataclasses.py [hunk 1]
```diff
@@ -869,26 +869,6 @@ class X(Y):
...
-        foo: ArbitraryType
-        bar: List[ArbitraryType]
-
-    class TestModel(BaseModel):
-        a: ArbitraryType
-        b: Test
-
-        model_config = ConfigDict(arbitrary_types_allowed=True)
-
-    TestModel(a=ArbitraryType(), b=(ArbitraryType(), [ArbitraryType()]))
```
Comment (adriangb) [inline tests/test_dataclasses.py:889]: I'm pretty sure we decided this is not going to be supported anymore

**Comment by adriangb** [comment]
please review

**Review by dmontagu — APPROVED** [review]
LGTM other than I can't independently confirm that we agreed to drop support for parsing tuples as dataclasses. Feel free to merge

**Comment by adriangb** [comment]
That's what I recall, if wrong let me know @samuelcolvin and I'll revert

**Pull request merged**